### PR TITLE
Remove VS Tools for Docker doc

### PR DESCRIPTION
**Pull request by scottaddie**
- Remove the VS Tools for Docker doc from the dotnet/docs repo.
- Point to the version which was migrated to the aspnet/Docs repo: https://docs.microsoft.com/aspnet/core/publishing/visual-studio-tools-for-docker

### CODE REVIEW

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @scottaddie. Can you also add the entry to the master redirection file: https://github.com/dotnet/docs/blob/master/.openpublishing.redirection.json?

**Comment by scottaddie** [comment]
@mairaw Nice catch! This is done.

File: docs/toc.md [hunk 1]
```diff
@@ -91,7 +91,7 @@
 ### [Runtime package store](core/deploying/runtime-store.md)
 ## [Docker](core/docker/index.md)
 ### [Building Docker Images for .NET Core Applications](core/docker/building-net-docker-images.md)
-### [Visual Studio Tools for Docker](core/docker/visual-studio-tools-for-docker.md)
+### [Visual Studio Tools for Docker](/aspnet/core/publishing/visual-studio-tools-for-docker.md)
```
Comment (mairaw) [inline docs/toc.md]: this link is broken. the md extension should be removed

File: docs/core/docker/index.md [hunk 2]
```diff
@@ -17,6 +17,6 @@ ms.assetid: cae32148-aef4-4e64-a7f0-88072bad4400
 The following tutorials are available for learning about using Docker with .NET Core.
 
 - [Building Docker Images for .NET Core Applications](building-net-docker-images.md)
-- [Visual Studio Tools for Docker](visual-studio-tools-for-docker.md)
+- [Visual Studio Tools for Docker](/aspnet/core/publishing/visual-studio-tools-for-docker.md)
```
Comment (mairaw) [inline docs/core/docker/index.md]: broken link. remove the .md extension

File: .openpublishing.redirection.json [hunk 3]
```diff
@@ -25,6 +25,10 @@
             "redirect_url": "/dotnet/core/deploying/index"
         },
         {
+            "source_path": "docs/core/docker/visual-studio-tools-for-docker.md",
+            "redirect_url": "/aspnet/core/publishing/visual-studio-tools-for-docker"
```
Comment (mairaw) [inline .openpublishing.redirection.json]: do you wanna move the comments with the article or make them disappear?

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM. A few broken links to fix before merging. Also had a question for you about the LiveFyre comments.

File: .openpublishing.redirection.json (hunk 3 above)
Comment (scottaddie) [reply .openpublishing.redirection.json]: If possible, I'd like to move the comments. Please advise on how this is done.

**Comment by scottaddie** [comment]
@mairaw Broken links fixed.

File: .openpublishing.redirection.json (hunk 3 above)
Comment (mairaw) [reply .openpublishing.redirection.json]: Sure! Then add the redirect_document_id like some other entries have.

Comment (scottaddie) [reply .openpublishing.redirection.json]: This is done. Thanks for the help!

**Review by mairaw — APPROVED** [review]
LGTM.

**Pull request merged**